### ISSUE TITLE
Page Title and short URL on separate lines?

### ISSUE THREAD

**Issue by RandH1607** [fwenzel/copy-shorturl#143]
Is there anyway in Firefox to have the copied page title and short URL on separate lines such as:

A very helpful page with a long URL
https://short-link/xxxyyyzzz

Or to put it another way to have a line break rather than a space between them.

If not could a selector for such an option be included on the "options" tab in the plug-in manager?

Much appreciated in advance and in hope.

**Comment by fwenzel** [comment]
Yeah I can do that - not on Christmas, but that seems reasonable! Glad you enjoy the add-on!

**Comment by RandH1607** [comment]
Thanks for the positive response.

Of course not over the holidays, but it is encouraging to know that you like the suggestion and it now on your to do list.

Yes, it is a very helpful plug-in.

Enjoy the holiday.
